From: chrysn <chrysn@fsfe.org>
Date: Sun, 27 Nov 2016 12:44:23 +0100
Subject: UIC gettext interoperability

Recent UIC versions seem to have changed their ways of invoking custom
helper functions. This is a quick fix, better workarounds are in
discussion with upstream.

Forwarded: https://github.com/openscad/openscad/issues/1872
---
 openscad.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/openscad.pro b/openscad.pro
index dc200bc..3b781a6 100644
--- a/openscad.pro
+++ b/openscad.pro
@@ -251,7 +251,7 @@ RESOURCES = openscad.qrc
 # Qt5 removed access to the QMAKE_UIC variable, the following
 # way works for both Qt4 and Qt5
 load(uic)
-uic.commands += -tr _
+uic.commands += -tr '"([](const char *a, void __attribute__((unused)) *b) {return QString(_(a));})"'
 
 FORMS   += src/MainWindow.ui \
            src/Preferences.ui \
